cmd/iacinventorycmd/iacinventorycmd.go

```diff
@@ -0,0 +1,53 @@
+package iacinventorycmd
+
+import (
+	"bytes"
+
+	"github.com/soluble-ai/soluble-cli/pkg/iacinventory"
+	"github.com/soluble-ai/soluble-cli/pkg/options"
+	"github.com/spf13/cobra"
+)
+
+func Command() *cobra.Command {
+	var username string
+	var oauthToken string
+	var submit bool
+	opts := options.PrintClientOpts{}
```

Comment:
So PrintClientOpts{} should be used when the command needs to print something back to the user.  It supports all sorts of things: tabular output, filtering, sorting, yaml/json output, etc.

ClientOpts{} just gives you an API client.

You can see the difference if you run the command with --help.  There's lots of printing-related options available for PrintClientOpts{}.